provider/machine_pool_resource.go

```diff
@@ -142,6 +151,29 @@ func (t *MachinePoolResourceType) GetSchema(ctx context.Context) (result tfsdk.S
 				},
 				Optional: true,
 			},
+			"availability_zones": {
+				Description: "The availability zones in which to place the machine pool",
+				Type: types.ListType{
+					ElemType: types.StringType,
+				},
+				Optional: true,
+				Computed: true,
+				PlanModifiers: []tfsdk.AttributePlanModifier{
+					tfsdk.UseStateForUnknown(),
+					ValueCannotBeChangedModifier(),
+				},
+			},
+			"subnet_ids": {
+				Description: "The subnet IDs in which to place the machine pool",
```

Comment:
I think we would like to be align with rosa cli. The description in rosa cli is 
"Select subnet to create a single AZ machine pool for BYOVPC cluster"